### PR TITLE
LPS-143278 Refining the DXP Cloud Setup Step to Finalize

### PR DESCRIPTION
**Pull request by jairmedeiros**

### CODE REVIEW

**Comment by jairmedeiros** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:test:relevant

**Comment by jairmedeiros** [comment]
ci:test:stable

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SetupDXP/index.js [hunk 1]
```diff
@@ -20,39 +20,25 @@ import { actionTypes } from '../../context/reducer';
...
-					)
-					.join('')}) and (contains(name, 'HA DR') or contains(name, 'Std DR'))`,
-				koroneikiAccountsFilter: `accountKey eq '${project.accountKey}'`
-			}
-		})
-	// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [project, userAccount]);
+	const { data } = useQuery(getSetupDXPInfo, {
+		variables: {
+			accountSubscriptionsFilter: `(accountKey eq '${project.accountKey}') and (contains(name, 'HA DR') or contains(name, 'Std DR'))`,
```
Comment (joseabelenda) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SetupDXP/index.js:29]: @jairmedeiros , can we move these strings 'HA DR' and 'Std DR' to another place instead hard code?
Does it make sense?

Comment (jairmedeiros) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SetupDXP/index.js:29]: Using Object or Picklist? I can pass these strings to consts variables, but still hard coded.

**Comment by joseabelenda** [comment]
resent:https://github.com/pt-liferay-solutions/liferay-portal/pull/445

**Pull request closed without merging**